src/schemas.py

```diff
@@ -10,3 +10,16 @@ class Project(BaseModel):
 
 class ProjectDetails(Project):
     project_id: uuid.UUID = Field(default_factory=uuid.uuid4)
+
+
+class User(BaseModel):
+    user_id: uuid.UUID = Field(default_factory=uuid.uuid4)
+    name: str
+    email: str
+    password: str
```

Comment:
add specific data types to validate email, have constrains on password too in terms of length, etc
And ofc in real app you 99% wont have password in database like that. it is usually managed by auth app